Automattic/pocket-casts-android
# Migrating RxJava to Coroutines

## Description

This PR migrates `emailChange` on the `SyncManager` to implement Coroutines.  Since the function was returning a `Single`, I migrated to use a suspend function. 
Also on the `ChangeEmailViewModel` , I changed `changeEmailState` to use a stateflow instead of livedata. I did this change since its a good step in the right direction to implement coroutines. 
I would be glad to proceed with more migrations to coroutines if this PR is accepted. 

Fixes # 
N/A
## Testing Instructions
1. Go to Profile 
2. Tap on Account
3. Navigate to Change Email Address
4. Confirm all inputs work as before just better this time with Coroutines 😄 

## Screenshots or Screencast 


## Checklist
 
#### I have tested any UI changes...
